.eslintrc.cjs

```diff
@@ -15,9 +15,7 @@ const config = {
     },
   ],
   parser: "@typescript-eslint/parser",
-  parserOptions: {
-    project: path.join(__dirname, "tsconfig.json"),
-  },
+  parserOptions: {},
```

Comment:
@Lada496 I didn't exactly remember why I deleted parserOptions (probably to solve an error happened before), but yeah you're right! I'll add parserOptions as you suggested.

Reply:
@keento0809 Hope you can still see this comment. This is the original suggestion that I wanted to update.
instead of empty object, which I remember the inside of override, and just put ignorePatterns outside of override. 
That should be it.
I think the original one was set by t3 so I assume they are correct (maybe lol)